error-handling/erros.js

```diff
@@ -1,4 +1,20 @@
-exports.handle500s = (err, req, res, next) => {
+exports.handle404s = (err, req, res, next) => {
   console.log(err);
-  res.status(500).send({ msg: "Internal Server Error" });
+  res.status(404).send({ msg: "Not Found" });
+};
+
+exports.idDoesNotExistError = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(404).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
+
+exports.invalidIdError = (err, req, res, next) => {
```

Comment:
this will handle other errors that PSQL sends back, how about handleSQLErrors to make it more generic?